src/order/filter/germ.lean

```diff
@@ -12,8 +12,29 @@ import algebra.pi_instances
 The germ of a function `f : α → β` at a filter `l : filter α` is the equivalence class of `f`
 with respect to the equivalence relation `eventually_eq l`: `f ≈ g` means `∀ᶠ x in l, f x = g x`.
 
-We define `mk l f` to be the germ of `f` at `l` and `const l c` to be the germ of the constant
-function at `l`.
+## Main definitions
+
+We define
+
+* `germ l β` to be the space of germs of functions `α → β` at a filter `l : filter α`;
+* coercion from `α → β` to `germ l β`: `(f : germ l β)` is the germ of `f : α → β`
+  at `l : filter α`; this coercion is declared as `has_coe_t`, so it does not require an explicit
+  up arrow `↑`;
+* coercion from `β` to `germ l β`: `(↑c : germ l β)` is the germ of the constant function
+  `λ x:α, c` at a filter `l`; this coercion is declared as `has_lift_t`, so it requires an explicit
+  up arrow `↑`;
```

Comment:
This is somewhat mysterious. Do we have documentation about `has_coe_t` and `has_lift_t` we could refer to?

Reply:
```suggestion
  up arrow `↑`, see [TPiL Sec. 10.6](https://leanprover.github.io/theorem_proving_in_lean/type_classes.html#coercions-using-type-classes)
  for details.
```